src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    private final Ball ball1;
+    private final Ball ball2;
+    private final Ball ball3;
+
+    public Lottery() {
+        this.ball1 = getRandomBall();
+        this.ball2 = getRandomBall();
+        this.ball3 = getRandomBall();
+    }
+
+    public Ball getBall1() {
+        return ball1;
+    }
+
+    public Ball getBall2() {
+        return ball2;
+    }
+
+    public Ball getBall3() {
+        return ball3;
+    }
+
```

Comment:
balls shoudln't be fields - that's a very fragile pattern, bad practices, not extendable at all. What if we needed to generate 4 balls? you would need to add more code. With correct approach, you would just change one number 3, to 4. Remove all of this